### PR TITLE
Fix modulesync; Drop EoL CentOS 6 support

### PR DESCRIPTION
**Pull request by h-haaks**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -11,15 +11,15 @@
   "dependencies": [
     {
       "name": "puppet/archive",
-      "version_requirement": ">= 1.0.0 < 6.0.0"
+      "version_requirement": ">= 1.0.0 < 5.0.0"
```
Comment (h-haaks) [inline metadata.json:14]: puppet/mysql_java_connector does not support 5.x

File: metadata.json [hunk 2]
```diff
@@ -11,15 +11,15 @@
...
     },
     {
       "name": "puppet/mysql_java_connector",
       "version_requirement": ">= 3.0.2 < 5.0.0"
     },
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.13.1 < 9.0.0"
+      "version_requirement": ">= 4.13.1 < 7.0.0"
```
Comment (h-haaks) [inline metadata.json:22]: puppet/mysql_java_connector does not support puppetlabs/stdlib >= 7.x

File: spec/acceptance/1_default_parameters_spec.rb [hunk 3]
```diff
@@ -1,50 +1,38 @@
...
-        target          => $jh,
-        fetch_options   => '-q -c --header "Cookie: oraclelicense=accept-securebackup-cookie"',
-        url             => #{java_url},
-        download_timout => 1800,
-        strip           => true,
+      $jh = '/opt/java'
+      file { $jh:
+        ensure => directory,
+      } ->
+      archive { 'jdk11':
```
Comment (h-haaks) [inline spec/acceptance/1_default_parameters_spec.rb:16]: Switch to Archive and zulu jdk8

File: spec/acceptance/1_default_parameters_spec.rb [hunk 4]
```diff
@@ -1,50 +1,38 @@
...
+        extract_path    => $jh,
+        creates         => "${jh}/bin/java",
+        cleanup         => true,
       } ->
       class { 'confluence':
-        version             => '5.5.6',
-        download_url        => #{download_url},
-        javahome            => $jh,
+        version      => '5.5.6',
+        jvm_type     => 'oracle-jdk-1.8',
```
Comment (h-haaks) [inline spec/acceptance/1_default_parameters_spec.rb:27]: Added this because jdk11 is the default but confluence 5.5.6 does not run on java11 ...

**Comment by h-haaks** [comment]
@bastelfreak is this ready for merge now?

File: metadata.json [hunk 5]
```diff
@@ -32,14 +32,12 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "6",
```
Comment (h-haaks) [inline metadata.json:35]: Acceptance tests failed on CentOS 6. Removed support since it's EOL

**Comment by h-haaks** [comment]
Fixes #211

**Comment by h-haaks** [comment]
I guess it also fixes #202

File: metadata.json (hunk 1 above)
Comment (h-haaks) [reply metadata.json:14]: I'll start working on modulesync issues in puppet/mysql_java_connector so that we can get this inline as well.

Comment (h-haaks) [reply metadata.json:14]: see https://github.com/voxpupuli/puppet-mysql_java_connector/pull/77

Comment (bastelfreak) [reply metadata.json:14]: thanks for all the work. I made a new release for the mysql_java_connector.

**Review by bastelfreak — APPROVED** [review]

**Comment by bastelfreak** [comment]
thanks for all the work!

**Pull request merged**